packages/horizontal-slider-react/src/HorizontalSlider.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect, useMemo, useRef } from 'react';
+
+let lastHorizontalSliderId = 0;
+
+function generateHorizontalSliderId() {
+  return `uic-horizontal-slider-${lastHorizontalSliderId++}`;
+}
+
+function cx(...classNames: Array<string | undefined>) {
+  return classNames.filter(Boolean).join(' ');
+}
+
+type RecordWithObjectID<TObject = {}> = TObject & {
+  objectID: string;
+};
+
+type HorizontalSliderClassnames = {
+  item: string;
+  list: string;
+  navigation: string;
+  navigationNext: string;
+  navigationPrevious: string;
+  root: string;
```

Comment:
```suggestion
  item?: string;
  list?: string;
  navigation?: string;
  navigationNext?: string;
  navigationPrevious?: string;
  root?: string;
```

with `const { classNames = {} } = props;` L#44 would avoid having the `props.classNames?.` syntax

Reply:
See 4b843ec6ba26b055ee4477af7f65aabd04590576.